### DISCUSSION TITLE
Package Resolution Failed

### DISCUSSION THREAD

**Discussion by jmontana** [orchetect/MenuBarExtraAccess#16] [Q&A] [answered]
Hi, trying to add this package to my project. When I do, I get:

MenuBarExtraAccess could not be resolved:
v MenuBarExtraAccess
  v Fetching from https://github.com/orchetect/MenuBarExtraAccess (cached)
  Couldn't get the list of tags:
  unexpectedly did not find the new dependency in the package graph: sourceControl(identity: menubarextraaccess, location: SwiftPM.SPMPackageDepend...

<img width="1089" alt="image" src="https://github.com/user-attachments/assets/3e39c0dc-121a-41b4-a8b8-06490160b884">

**Comment by orchetect** [comment, marked as answer]
That's an Xcode bug that happens spuriously. You can 'Add Anyway' and then quit Xcode and reopen the project.

You might also need to Update to Latest Package Versions and/or Reset Package Caches. Worst case, quit Xcode then delete your DerivedData folder.

**Reply by jmontana** [reply]
Thanks! I had to random combinations of all of those options, but eventually it worked.
